ZenPacks/zenoss/OpenStackInfrastructure/Instance.py

```diff
@@ -47,7 +47,7 @@ def set_host_name(self, hostname):
         if len(hypervisors):
             hypervisor = sorted(hypervisors)[0]
             if len(hypervisors) > 1:
-                LOG.error("Multiple hypervisors were found matching hostname %s - Choosing %s" %
+                LOG.warn("Multiple hypervisors were found matching hostname %s - Choosing %s" %
```

Comment:
Why the change?   This is something that should not be happening, and it's likely to cause problems if it is- that's why it was an error. 

Were you seeing this occur?